logparser panic on reference to file in directory with restricted permissions

```
2017-06-11T17:46:06Z E! Error in plugin [inputs.logparser]: open /var/log/supervisor/my_listener-stdout.log: permission denied
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0xbd861d]

goroutine 51 [running]:
github.com/influxdata/telegraf/plugins/inputs/logparser.(*LogParserPlugin).receiver(0xc42012a080, 0x0)
	/home/ubuntu/telegraf-build/src/github.com/influxdata/telegraf/plugins/inputs/logparser/logparser.go:181 +0x6d
created by github.com/influxdata/telegraf/plugins/inputs/logparser.(*LogParserPlugin).tailNewfiles
	/home/ubuntu/telegraf-build/src/github.com/influxdata/telegraf/plugins/inputs/logparser/logparser.go:167 +0x416
```

### System info:

Telegraf v1.3.1 (git: release-1.3 f93615672b02a41d9bc867bd92bf31c1d777989b)

Amazon Linux

### Steps to reproduce:

Add a logparser entry to a file within a directory with chmod 600 and different ownership.

--- a/plugins/inputs/logparser/logparser.go
+++ b/plugins/inputs/logparser/logparser.go
@@ -182,7 +182,10 @@ func (l *LogParserPlugin) tailNewfiles(fromBeginning bool) error {
 					Location:  &seek,
 					MustExist: true,
 				})
-			l.acc.AddError(err)
+			if err != nil {
+				l.acc.AddError(err)
+				continue
+			}
 
 			// create a goroutine for each "tailer"
 			l.wg.Add(1)

